src/components/views/rooms/SendMessageComposer.tsx

```diff
@@ -497,7 +498,7 @@ export class SendMessageComposer extends React.Component<ISendMessageComposerPro
 
     private get editorStateKey() {
         let key = `mx_cider_state_${this.props.room.roomId}`;
-        if (this.props.relation?.rel_type === RelationType.Thread) {
+        if (this.props.relation?.rel_type === THREAD_RELATION_TYPE.name) {
```

Comment:
Shouldn't this use `THREAD_RELATION_TYPE.matches(relation?.rel_type)` to support both legacy and new?